src/internal/packager/sbom/catalog.go

```diff
@@ -153,7 +159,7 @@ func (b *Builder) createImageSBOM(tag name.Tag) ([]byte, error) {
 	}
 
 	// Write the sbom to disk using the image tag as the filename
-	sbomFile, err := b.createSBOMFile("%s.json", tag.String())
+	sbomFile, err := b.createSBOMFile(fmt.Sprintf("%s.json", tag.String()))
```

Comment:
Nit. Most places in code we try to avoid `fmt.Sprintf()` inline, especially if it has to call a function too.